test/oph/ehoks/palaute/opiskelija_test.clj

```diff
@@ -127,14 +127,14 @@
         (testing
          "initiate aloituskysely if `osaamisen-hankkimisen-tarve` is `true`."
           (is (= (op/initial-palaute-state-and-reason
-                   ctx {:type :aloituskysely})
+                   ctx :aloituskysely nil)
```

Comment:
Tän ehkä pitäis olla `[]` eikä `nil`.  Yritän tarkistaa ottiko just tää funktio listan vai yhden. :)

Reply:
ottaa monta.  Ei pitäisi olla `nil`.

Reply:
vakavuus: pyyntö

Reply:
```suggestion
                   ctx :aloituskysely [])
```

Reply:
Mutta vielä parempi jos saadaan muutetuksi tuon parametrilista siten, että:

```suggestion
                   (assoc ctx :existing-heratteet []) :aloituskysely)
```

Reply:
Mä itseasiassa uudelleennimesin tuon opiskelijapalautteiden `existing-heratteet!` -> `existing-palaute!` ja laitoin sen palauttamaan yhden palautteen, koska sieltä joka tapauksessa palautuu (ainakin pitäisi palautua) aina yksi palaute. :) `:existing-palaute` on uudessa koodissa osa `ctx`-mappia.